vaadinfrontend/src/main/java/life/qbic/datamanager/views/projects/project/experiments/experiment/components/ExperimentalVariablesComponent.java

```diff
@@ -27,61 +24,36 @@
  */
 public class ExperimentalVariablesComponent extends Card {
 
-  private boolean controlsEnabled = true;
-  private final List<Component> variableFactSheets = new ArrayList<>();
   @Serial
   private static final long serialVersionUID = 7589385115005753849L;
   private final Collection<ExperimentalVariable> experimentalVariables;
-  private final MenuBar menuBar;
   private final Div controls = new Div();
   private final Div content = new Div();
+  private final Button editButton;
   private final CreationCard variableCreationCard;
 
 
   private final List<ComponentEventListener<ExperimentalVariablesEditEvent>> listeners = new ArrayList<>();
-
   private final List<ComponentEventListener<AddNewExperimentalVariableEvent>> listenersNewVariable = new ArrayList<>();
+  private final List<Div> variableFactSheets = new ArrayList<>();
 
   private ExperimentalVariablesComponent(Collection<ExperimentalVariable> experimentalVariables) {
     this.experimentalVariables = experimentalVariables;
-    this.menuBar = createMenuBar();
+    this.editButton = createEditButton("Edit");
     variableCreationCard = CreationCard.create("Add experimental variables");
     variableCreationCard.addListener(event -> fireAddEvent());
     layoutComponent();
+    configureComponent();
   }
 
-  private MenuBar createMenuBar() {
-    MenuBar menu = new MenuBar();
-    menu.addThemeVariants(MenuBarVariant.LUMO_TERTIARY);
-    MenuItem menuItem = menu.addItem("•••");
-    SubMenu subMenu = menuItem.getSubMenu();
-    subMenu.addItem("Add", event -> fireAddEvent());
-    subMenu.addItem("Edit", event -> fireEditEvent());
-    return menu;
+  private static Button createEditButton(String label) {
+    Button button = new Button(label);
+    return button;
   }
```

Comment:
why do you need a method for `new Button(String)`? I suggest inlining this method.

Reply:
nice spot